docs/developer/putting-it-together.md

```diff
@@ -0,0 +1,84 @@
+Your ideal delivery blueprint spans delivery flow, generators, editors
+and other commands. All we need is something to pull it together.
+
+Your event listeners need to be invoked by Atomist handlers. The
+`SoftwareDeliveryMachine` takes care of this, ensuring that the
+correct handlers are emitted for use in `atomist.config.ts`, without
```

Comment:
wait. We eliminated atomist.config.ts in favor of index.ts

Reply:
I'm not sure; this text (as with most of the new content) was just taken [directly from what existed](https://github.com/atomist/docs/pull/286/files#diff-cacc5a8626a2cc71344704d473c67b79L442). I think you're probably right and this just hadn't been updated.